Networking/Poller.cc

```diff
@@ -154,6 +154,11 @@ namespace litecore { namespace net {
     }
 
 
+    void Poller::interrupt(int fd) {
```

Comment:
Note: I added an assertion here since passing a non-positive FD would have a bad effect. That necessitated splitting out the guts into `_interrupt` since Poller itself does need to pass negative numbers or zero.